scripts/l2/l2-op-node-restart.sh

```diff
@@ -15,6 +15,24 @@ docker compose -f docker/docker-compose-l2.yml stop op-node
 # set L2OO or DGF env vars
 post_deployment_setup_env_vars $(pwd)/.deploy/op-devnet-deployments-${L2_CHAIN_ID}.json $DEVNET_L2OO
 
+ROLLUP_CONFIG=$(pwd)/.deploy/rollup.json
+# set babylonFinalityGadgetRpc in rollup.json
+if [ "$BBN_FINALITY_GADGET_RPC" != "" ]; then
```

Comment:
<img width="948" alt="image" src="https://github.com/user-attachments/assets/10016d73-1774-411c-a8cd-9747382062c2">

seems we can do ^

Reply:
actually we don't need this check at all

l2-op-node-restart.sh should support case where we want to disable FG

i.e. setting it from non-empty to empty

we just override whatever it has in the .env to rollup.json

Reply:
yeah, it is updated.